spec/driver_spec.cr

```diff
@@ -30,13 +30,28 @@ describe Driver do
     DB.open "mysql://crystal_test:secret@#{database_host}/crystal_mysql_test" do |db|
       db.scalar("SELECT DATABASE()").should eq("crystal_mysql_test")
       db.scalar("SELECT CURRENT_USER()").should match(/^crystal_test@/)
+      db.scalar("SELECT @@collation_connection").should eq("utf8_general_ci")
```

Comment:
I added these 2 assertions to check when the encoding is not provided in the connection string, the DB connection's collation should be utf8_general_ci.

Not sure if it's a good practice to have that. I assume the DB server could have a different collation set in the config.ini

Reply:
I added another section to the README.md